kubernetes/go/'

```diff
@@ -0,0 +1,28 @@
+#--------------------------------------------------------------------
```

Comment:
Was this accidentally added to the repo?

Reply:
good catch!